stylelint.config.js

```diff
@@ -15,7 +15,6 @@ module.exports = {
     'max-nesting-depth': 3,
     'no-extra-semicolons': [true, { severity: 'warning' }],
     'scss/at-mixin-pattern': '^[a-z]+([a-z0-9-]+[a-z0-9]+)?$',
-    'suitcss/custom-property-no-outside-root': true,
```

Comment:
We can probably remove the suitcss plugin all together. For passively we might need to keep it until the next major version, but it looks like this was the only rule we were using from that plugin.

Reply:
I'll go ahead and remove it. I don't think it's non-passive.